src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivitiesResourceIntTest.kt

```diff
@@ -376,29 +372,14 @@ class ActivitiesResourceIntTest : IntegrationTestBase() {
     private val createOffenderProgramProfileRequest: () -> CreateOffenderProgramProfileRequest = {
       CreateOffenderProgramProfileRequest(
         bookingId = repository.lookupOffender("A1234TT")?.latestBooking()?.bookingId!!,
-        startDate = LocalDate.parse("2022-10-31"),
-        endDate = LocalDate.parse("2022-11-30"),
+        startDate = LocalDate.parse("2022-11-14"),
+        endDate = LocalDate.parse("2022-11-21"),
```

Comment:
Made these dates different to the activity schedule start and end date to avoid confusion.